Library/Formula/squid-deb-proxy.rb

```diff
@@ -0,0 +1,170 @@
+class SquidDebProxy < Formula
+  desc "APT repo caching proxy based on Squid."
+  homepage "https://launchpad.net/squid-deb-proxy"
+
+  url "http://archive.ubuntu.com/ubuntu/pool/universe/s/squid-deb-proxy/squid-deb-proxy_0.8.14.tar.gz"
+  sha256 "d30aa9d8185c1e424608deb307f8e92aee8270acccc3c0b26af1d03a2feae92b"
+
+  depends_on "squid"
+
+  # assorted changes that should be submitted upstream
+  patch :DATA
+
+  def install
+    # fix up issues specific to Homebrew
+    inreplace "squid-deb-proxy.conf" do |s|
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! %r{([\s='"])/etc/}, "\\1#{etc}/"
+      s.gsub! %r{([\s='"])/var/}, "\\1#{var}/"
+    end
+
+    inreplace "init-common.sh" do |s|
+      # install should be as the calling user
+      s.gsub! /(\binstall\b.*?)(\s+-[og]\s*\S+)+/, "\\1"
+
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! "/usr/sbin/squid", "#{HOMEBREW_PREFIX}/opt/squid/sbin/squid"
```

Comment:
Changing this to `#{sbin}/squid` makes the resulting path `/usr/local/Cellar/squid-deb-proxy-0.8.14/sbin/squid`, which is not where `squid` lives. The `squid` binary does not come from this formula; it's inherited from the `squid` formula, thus the dependency.